src/interceptors/fetch/index.ts

```diff
@@ -46,13 +47,27 @@ export class FetchInterceptor extends Interceptor<HttpRequestEventMap> {
 
       this.logger.info('awaiting for the mocked response...')
 
+      const signal = interactiveRequest.signal
+      const requestAborted = new DeferredPromise()
+
+      signal.addEventListener(
+        'abort',
+        () => {
+          requestAborted.reject(signal.reason)
+        },
+        { once: true }
```

Comment:
I've made sure we only react to the abort event once to make sure there's no event emitter memory leak when cloning requests (each request clone will also preserve previously attached listeners, I believe; I recall an issue about this in the past).